extensions/wikia/UserLogin/js/UserSignup.js

```diff
@@ -110,9 +110,9 @@
 
 			if (
 				window.wgUserLoginDisableCaptcha !== true &&
-				inputs.recaptcha_response_field // jshint ignore:line
+				inputs['g-recaptcha-response']
```

Comment:
JS can't handle dot syntax when there's dashes in the key. Otherwise, yes, dot syntax is preferred.